quilt/test/test_command.py

```diff
@@ -149,7 +150,7 @@ def _mock_logs_list(self, owner, package, pkg_hash):
         logs_url = "%s/api/log/%s/%s/" % (command.QUILT_PKG_URL, owner, package)
         resp = dict(logs=[dict(
             hash=pkg_hash,
-            created=str(datetime.now()),
+            created=str(datetime.now().strftime(UTCF)),
```

Comment:
That's actually going to be a local time, not UTC (not that it really matters in a unit test)